cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -157,25 +157,14 @@ features:
     Configure:
       title: Configure
       buttons:
-        add-ns: Add Namespace
```

Comment:
these are not needed?

Reply:
Yep, these are not needed after we revamped the Admin page.